server/src/utils/csv2GeojsonUtils.ts

```diff
@@ -0,0 +1,49 @@
+import * as fs from "fs";
+import * as path from "path";
+import csvtojson from "csvtojson";
+
+const createDirectoryIfNotExists = (directory: string) => {
+    if (!fs.existsSync(directory)) {
+        fs.mkdirSync(directory, { recursive: true });
+    }
+};
+
+async function convertCSVToGeoJSON(csvFilePath: string) {
+    try {
+        const geojsonDir: string = path.resolve(__dirname, "..", "data", "csv_conversion");
+        // const fileName: string = path.basename(csvFilePath, path.extname(csvFilePath));
+        const geojsonFileName: string = `points.geojson`;
+        const geojsonFilePath: string = path.resolve(geojsonDir, geojsonFileName);
+
+        createDirectoryIfNotExists(geojsonDir);
+
+        const jsonObj: any[] = await csvtojson().fromFile(csvFilePath);
```

Comment:
variable names need to be more verbose. jsonObj is not a good naming convention.